src/test/java/seedu/address/model/equipment/UniqueWorkListListTest.java

```diff
@@ -0,0 +1,108 @@
+package seedu.address.model.equipment;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+//import static seedu.address.logic.commands.CommandTestUtil.VALID_ADDRESS_BOB;
+//import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_HUSBAND;
+import static seedu.address.testutil.TypicalEquipments.ACHORVALECC;
+import static seedu.address.testutil.TypicalEquipments.BOB;
+
+//import java.util.Arrays;
+//import java.util.Collections;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import seedu.address.model.equipment.exceptions.DuplicateEquipmentException;
+import seedu.address.model.equipment.exceptions.EquipmentNotFoundException;
+//import seedu.address.testutil.EquipmentBuilder;
+
+public class UniqueWorkListListTest {
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    private final WorkList typicalWorkListA = new WorkList("2019-01-01", "Alice");
```

Comment:
Shouldn't have Alice anymore. Alice is changed to Anchorvale CC name
I havent changed bob so it should still be fine.